### DISCUSSION TITLE
copy pictures from clipboard?

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#42] [Q&A] [answered]
Hi Renaud,

Is it possible to copy pictures from clipboard to a component in Dlite?
I tried Picture, TextArea, Caroussel but I could not get it to work.
I will use a button to show or hide the picture.

Folkert

**Comment by HFolkertH** [comment]
hi Renaud

I managed to get a picture  in a Text component <img src=encode 64....>.  I used https://www.base64-image.de
But much better would be to get it inside a Picture component.

**Reply by renaudpawlak** [reply]
Well, there is an ``Image`` component, which is made for it. However it does not support pasting images because the idea was to reference external images through Internet, in order to avoid having huge project files (*.dlite files). It would not be very good for eco-design ;)

However, for obvious reasons, I can imagine that it would be very convenient to be able to copy/paste images that would be embedded in the project directly without having to deploy the images. I could impose a limit in the image size to avoid wasting of resources. And I need to monitor it so that the user would be aware of what is used by the app.

I am creating an issue to take it into account.

**Reply by HFolkertH** [reply]
👍
duim omhoog
Unicode: U+1F44D, UTF-8: F0 9F 91 8D

**Comment by renaudpawlak** [comment, marked as answer]
Done.
